.gitignore

```diff
@@ -1,3 +1,4 @@
+/data
```

Comment:
Not needed anymore. I've changed the script to start mongodb using `/etc/mongod.conf`so it does not create that dir there anymore